Frontend/src/AddUserForm.svelte

```diff
@@ -0,0 +1,88 @@
+<script>
+    let title;
+    let firstName;
+    let lastName;
+    let email
+
+    let location = [];
+    let group = [];
+
+    const handleSubmit = () => {
+        console.log("Submitted")
+    }
+
+</script>
+
+<main>
+    <form on:submit|preventDefault={handleSubmit} class="modal-div"><!--  prevent default is used to stop the page from refresh -->
+        <label>
+            Title
+            <input type="text" placeholder="" bind:value={title}>
+        </label>
+    
+        <label>
+            First Name
+            <input type="text" placeholder="" bind:value={firstName}>
+        </label>
+        
+        <label>
+            Last Name
+            <input type="text" placeholder="" bind:value={lastName}>
+        </label>
+    
+        <label>
+            Email Address
+            <input type="text" placeholder="" bind:value={email}>
+        </label>
+    
+        <label>
+            Location(s)
+            <select bind:value={location}>
+                <option value="TEST 1">TEST</option>
+                <option value="TEST 2">TEST</option>
+                <option value="TEST 3">TEST</option>
+            </select>
+        </label>
+        <small>Create New</small>
+    
+        <label>
+            Group(s)
+            <select bind:value={group}>
+                <option value="TEST 1">TEST</option>
+                <option value="TEST 2">TEST</option>
+                <option value="TEST 3">TEST</option>
+            </select>
+        </label>
+        <small>Create New</small>
+    </form >
+</main>
+
+
+<style>
```

Comment:
It probably hasn't given you too much trouble yet (and maybe it even won't in this project), but my advice would be to keep your styles separate from your code. You can totally use the style tag to import them if you chose to have multiple css files, but once you get into the habit of putting little tweaks here and there in your components, you might have something later down the line that is hard to diagnose because your styles are scattered everywhere

Reply:
I was under the impression that was a feature of Svelte. As it allows you to make by component changes. So you recommend having it all in my styles folder?

Reply:
Nope! They're just html tags, but I get what you mean -especially when the official documentation uses them all the time. In general inline styles and wild style tags are difficult to manage and keep track of, regardless of library/framework

Reply:
I was thinking about this very topic today at work and I think \<style\> tags are fine to use, but just make sure you're well organized with your use of them. I was kind of thinking that a one-to-one svelte to css situation is basically the same situation, just in different places. Either way you go about it, just make sure that your system is intuitive and clean